tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -128,6 +129,8 @@ export class WebGPUBackend extends KernelBackend {
   private querySet: GPUQuerySet;
   private fromPixelProgram?: FromPixelsProgram;
   private fromPixelImportProgram?: FromPixelsImportProgram;
+  private deferredSubmitBatchSize =
+      env().get('WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE');
```

Comment:
I suppose `deferredSubmitBatchSize` is only initialized once. If we want to adjust `WEBGPU_DEFERRED_SUBMIT_BATCH_SIZE`'s value. `deferredSubmitBatchSize` won't be updated correspondingly. For example, in benchmarks, we change the webgpu deferred submit batch size, in current change, it may not take affect.

Reply:
I assume we don't needx to change this if fromPixel has been integrated into the unify program framework? I understand this is a quick try to see performance?